manifests/params.pp

```diff
@@ -48,33 +48,26 @@
   case $xfacts['lsbdistid'] {
     'debian', 'raspbian': {
       case $xfacts['lsbdistcodename'] {
-        'squeeze': {
-          $legacy_origin       = true
-          $origins             =  ['${distro_id} ${distro_codename}-security', #lint:ignore:single_quote_string_with_variables
-                                  '${distro_id} ${distro_codename}-lts',] #lint:ignore:single_quote_string_with_variables
-        }
-        'wheezy': {
-          $legacy_origin      = false
+        'squeeze', 'wheezy': {
+          $legacy_origin      = true
           $origins            = [
-            'origin=Debian,archive=oldoldstable,label=Debian-Security',
+            '${distro_id} ${distro_codename}-security', #lint:ignore:single_quote_string_with_variables
+            '${distro_id} ${distro_codename}-lts', #lint:ignore:single_quote_string_with_variables
```

Comment:
if these are facts, and they suspiciously look like facts, then i would address them as `$facts[name]`
but if the entire class, or module, addresses them that way, we can keep that for another pr

Reply:
Those aren't facts, they are macros from apt which should be like this in the config file.

Reply:
Those are not facts (although they look like) unattended upgrades uses variables instead of distribution names by default on its configuration, eg `$distro_codename` instead of `buster`

we could use facts and write `buster` but it wont be the most common way of configuring unattended upgrades (it will work anyway)

Reply:
aaaaaaaaah, hence the `'` single `'` quotes.